.github/workflows/docker_test.yml

```diff
@@ -0,0 +1,11 @@
+on: [pull_request]
+
+jobs:
+  docker_build:
+    runs-on: ubuntu-latest
+    name: Build Artifacts Docker Image
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Docker Image
+        run: docker build . -f Dockerfile
```

Comment:
Will this actually fail the action if problem occurs ?

Reply:
Yep, will have a non-zero exit code